apps/backend/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+services:
+  dynamodb-local:
+    command: '-jar DynamoDBLocal.jar -sharedDb -dbPath ./data'
+    image: 'amazon/dynamodb-local:latest'
+    container_name: dynamodb-local
+    ports:
+      - '8000:8000'
+    volumes:
+      - './docker/dynamodb:/home/dynamodblocal/data'
```

Comment:
I not sure, do `/home/dynamodblocal` your folder structure is in your computer? if true then it will not working on my computer

Reply:
The `/home/dynamodblocal` is not my folder structure. It in the docker destop (`docker container/docker images/home/data)